docs/docs/transpiler.mdx

```diff
@@ -6,6 +6,38 @@ description: The Transpiler | Transloco Angular i18n
 The transpiler is responsible for resolving the given value.  
 For example, when given `Hello {{ key }}` the default transpiler will replace the dynamic variable `key` based on the given params, or in some cases, within the [translation object itself](./additional-functionality#reference-other-keys).
 
+## DefaultTranspiler
```

Comment:
The documentation for the new property should be part of the [config options page](https://ngneat.github.io/transloco/docs/config-options/).